### ISSUE TITLE
What flavor of Markdown is supported?

### ISSUE THREAD

**Issue by StephenNay** [mrin9/RapiDoc#95]
I need to have a Markdown table in my documentation, and I wanted some of the columns to be centered. I tried the GFM style of using colons in the delimiter row, like this:

```
| Heading 1 | Left-Justified | Centered | Right-Justified |
| --------- | :------------- | :------: | --------------: |
| text | text | text | text |
```

The columns were all still left-justified, so apparently the Markdown engine doesn't support GFM. What Markdown flavor/tags are supported?

**Comment by mrin9** [comment]
It supports [CommonMark 0.29](https://spec.commonmark.org/0.29/) which is very near to GFM. 
The above syntax should work. the issue here is some style elements are getting overridden. I will fix it in our next patch.

> Next patch will be our last patch of the year 2019. So will wait for more issues to accumulate before releasing one.

**Comment by mrin9** [comment]
fixed in `7.1.0`
https://mrin9.github.io/RapiDoc/examples/markdown.html
